lighthouse-core/audits/byte-efficiency/render-blocking-resources.js

```diff
@@ -198,17 +209,15 @@ class RenderBlockingResources extends Audit {
     const {results, wastedMs} = await RenderBlockingResources.computeResults(artifacts, context);
 
     let displayValue = '';
-    if (results.length > 1) {
-      displayValue = `${results.length} resources delayed first paint by ${wastedMs}ms`;
-    } else if (results.length === 1) {
-      displayValue = `${results.length} resource delayed first paint by ${wastedMs}ms`;
+    if (results.length > 0) {
+      displayValue = i18n(UIStrings.displayValue, {timeInMs: wastedMs, itemCount: results.length});
     }
 
     /** @type {LH.Result.Audit.OpportunityDetails['headings']} */
     const headings = [
-      {key: 'url', valueType: 'url', label: 'URL'},
-      {key: 'totalBytes', valueType: 'bytes', label: 'Size (KB)'},
-      {key: 'wastedMs', valueType: 'timespanMs', label: 'Download Time (ms)'},
+      {key: 'url', valueType: 'url', label: i18n(i18nUtils.UIStrings.columnURL)},
```

Comment:
naming-wise, how about: `uiS(i18n.UIStrings.columnURL)`
bikeshedding wise, "i18n" feels most like a namespace so i hesitate to use it for a method name.

Reply:
chatted outside and went with `str_` for simplicity and easy future refactoring :)